modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ContainsFunction.java

```diff
@@ -60,4 +71,31 @@ protected Boolean apply(String string1, String string2) {
 		return string1.contains(string2);
 	}
 
+	private Boolean _apply(
+		JSONObject componentValuesJSONObject, JSONObject ruleValuesJSONObject) {
+
+		if ((componentValuesJSONObject == null) ||
+			(ruleValuesJSONObject == null)) {
+
+			return false;
+		}
+
+		Set<String> keySet = ruleValuesJSONObject.keySet();
+
+		for (String key : keySet) {
+			String value = componentValuesJSONObject.getString(key);
+
+			if (value != null) {
+				if (!Objects.equals(value, ruleValuesJSONObject.get(key))) {
+					return false;
+				}
+			}
+			else {
+				return false;
+			}
```

Comment:
@joseabelenda, how about...
```
if (value == null) {
	return false;
}

if (!Objects.equals(value, ruleValuesJSONObject.get(key))) {
	return false;
}
```
This is more similar to the code style of Liferay and hence what [Brian Chan](https://github.com/brianchandotcom/) usually prefers. 🙂 

I honestly hope you don't get angry, sad or disappointed with my comments, because my role here is only provide tips and guidance regarding the code style and, consequently, increase the chances of your pull be merged by Brian Chan without rework.

Reply:
That's ok!
I understand perfectly.
As new hired, I'm still learn the code pattern.
Thanks for all!